green_city/tests/api/get_eco_news_likes_test.py

```
@@ -0,0 +1,88 @@
+import pytest
+import requests
+import json
+from jsonschema.validators import validate
+
+from green_city.src.config import API_BASE_URL_8085, ENDPOINTS
+
+
+def test_get_eco_news_likes_count_success(create_news):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)